rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.66.1"
+channel = "1.68.0"
```

Comment:
No way to just use this file in `package-ci.yml` ?

Reply:
We use the action `dtolnay/rust-toolchain` in `package-ci` be it doesn't support `rust-toolchain.toml`

see dtolnay/rust-toolchain#77